src/pykeen/losses.py

```diff
@@ -53,6 +137,11 @@ def __init__(self, size_average=None, reduce=None, reduction: str = 'mean'):
 class PointwiseLoss(Loss):
     """Pointwise loss functions compute an independent loss term for each triple-label pair."""
 
+    @staticmethod
+    def validate_labels(labels: torch.FloatTensor) -> bool:
+        """Check whether labels are in [0, 1]."""
+        return labels.min() >= 0 and labels.max() <= 1
```

Comment:
would `(0 <= labels <= 1).all()` work?

Reply:
I am not sure :sweat_smile: For normal floating point this works, but I am not sure whether it also works for tensors.

Reply:
okay no harm in leaving it as is